From d0520df11fae7f106efcdce07c771168afa874a5 Mon Sep 17 00:00:00 2001
From: Madhu <enometh@net.meer>
Date: Sun, 20 Jun 2021 12:54:46 +0530
Subject: [PATCH 2/2] git-instaweb - preserve gitweb_config.perl if it exists

we don't want to overwrite the config file each time
especially since the doc says project_root can be set
---
 git-instaweb.sh | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/git-instaweb.sh b/git-instaweb.sh
index c68f49454c..5456d3871b 100755
--- a/git-instaweb.sh
+++ b/git-instaweb.sh
@@ -768,7 +768,8 @@ restart)
 	;;
 esac
 
-gitweb_conf
+#;madhu 210620 - dont overwrite each time
+test -f "$fqgitdir/gitweb/gitweb_config.perl" || gitweb_conf
 
 resolve_full_httpd
 mkdir -p "$fqgitdir/gitweb/$httpd_only"
-- 
2.35.1.dirty

